### DISCUSSION TITLE
If I Register With CLI, Then With The Android App, Is The CLI Reg Voided?

### DISCUSSION THREAD

**Discussion by MadManMoon** [AsamK/signal-cli#400] [Q&A] [answered]
Purely asking out of interest, but I'm just wondering if I need to re-register and make a master of my desktop signal again.

I definitely don't wish to use android signal if that is the master, I don't want that information being pumped around anywhere, no matter how secure it is. (yes, yes, I know I'm a contradictory soab ;-) )

I'll use the Windows client, and run cli on a pi, and maybe find another way to interact with the messages when 'abroad'.

**Comment by exquo** [comment, marked as answer]
Yes, there can only be one master device for each phone number. Any other devices that want to 'share' that account should be linked as 'slave' devices. Android app can only be master; signal-desktop can only be slave; signal-cli can be either.
